processing/src/main/java/io/druid/query/filter/DruidFloatPredicate.java

```diff
@@ -22,9 +22,31 @@
 /**
  * FloatPredicate is only supported in Java 8+, so use this to avoid boxing when a float predicate is needed.
  */
+@SuppressWarnings("unused")
 public interface DruidFloatPredicate
```

Comment:
Is this class an `ExtensionPoint` or not? If yes, please add annotation and Javadocs to methods.

Reply:
not an extension point.